EntityFramework.Exceptions.Tests/DatabaseTests.cs

```diff
@@ -7,10 +7,17 @@ namespace EntityFramework.Exceptions.Tests
     public abstract class DatabaseTests
     {
         internal DemoContext Context { get; }
+        internal DbContextOptions<DemoContext> ContextOptions { get; }
 
-        protected DatabaseTests(DemoContext context)
+        protected DatabaseTests(DbContextOptions<DemoContext> contextOptions)
         {
-            Context = context;
+            ContextOptions = contextOptions;
+            Context = GetNewContext();
+        }
+
+        protected DemoContext GetNewContext()
```

Comment:
Why is it protected?

Reply:
There is indeed no need, I changed it to private now. Thanks for pointing that out!